terraform/workspace-variables/review.tfvars

```diff
@@ -9,6 +9,6 @@ paas_app_start_timeout = "180"
 paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
-paas_web_app_memory = 1024
+paas_web_app_memory = 4096
```

Comment:
Will bump this back down once we've seeded the banding test data and we've got the ✅  from the rest of the team